src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -189,7 +193,11 @@ private void InitializeValues(HttpMethod method, Uri? requestUri)
             }
             if ((requestUri != null) && (requestUri.IsAbsoluteUri) && (!HttpUtilities.IsHttpUri(requestUri)))
             {
+#if TARGETS_BROWSER
```

Comment:
Ditto

Reply:
I like the more detailed one so left it in but moved the message definition to partial file as described.